### PR TITLE
Fixed style code

### PR DESCRIPTION
**Pull request by araslanov-e**

### CODE REVIEW

File: app/models/payment.rb [hunk 1]
```diff
@@ -12,6 +12,7 @@ class Payment < ActiveRecord::Base
   before_create :convert_and_save_usd_amount
 
   scope :accepted, -> { where(state: 'accepted') }
+  scope :in_current_month, -> { where(created_at: [Time.current.beginning_of_month..Time.current.end_of_month]) }
```
Comment (Mehonoshin) [inline app/models/payment.rb]: Let's cover this scope with specs

File: app/models/withdrawal.rb [hunk 2]
```diff
@@ -11,6 +11,8 @@ class Withdrawal < ActiveRecord::Base
 
   after_create :decrease_user_balance
 
+  scope :by_interval, ->(interval) { where('(DATE(?) - DATE(withdrawals.created_at)) < ?', Time.current, interval) }
```
Comment (Mehonoshin) [inline app/models/withdrawal.rb]: Let's add spec as well

File: app/models/payment.rb (hunk 1 above)
Comment (araslanov-e) [reply app/models/payment.rb]: It is testing in user_spec.rb

File: app/models/concerns/last_days_filterable.rb [hunk 3]
```diff
@@ -2,6 +2,8 @@ module LastDaysFilterable
   extend ActiveSupport::Concern
 
   included do
-    scope :in_days, ->(number_of_days) { where(created_at: [number_of_days.days.ago..Date.current.tomorrow]) }
```
[the remark this answers is not in the document]
Comment (araslanov-e) [reply app/models/concerns/last_days_filterable.rb]: Big scope

[the remark this answers is not in the document]
Comment (araslanov-e) [reply app/models/concerns/last_days_filterable.rb]: Big scope

File: app/models/withdrawal.rb (hunk 2 above)
Comment (araslanov-e) [reply app/models/withdrawal.rb]: It is testing user_spec.rb (non_paid_clients)

**Pull request merged**